packages/sw-build/src/lib/generate-sw.js

```diff
@@ -55,7 +55,7 @@ const errors = require('./errors');
  * @param {number} [input.maximumFileSizeToCacheInBytes] An optional number to
  * define the maximum file size to consider whether the file should be
  * precached. (Defaults to 2MB).
- * @param {Regex} [input.dontCacheBustUrlsMatching] An optional regex that will
+ * @param {Regexp} [input.dontCacheBustUrlsMatching] An optional regex that will
```

Comment:
I assumed it was RegExp but similarly don't know how well JSDoc would trip up on casing here. Deferring to Matt.